components/MechPage/GetInvolved.jsx

```diff
@@ -37,29 +38,38 @@ export const GetInvolved = () => (
     </h2>
 
     <div className="grid grid-cols-1 gap-8 md:grid-cols-3">
-      {list.map(({ title, desc, icon, url, urlName, isExternal }) => (
-        <div
-          key={title}
-          className="lg:p-6 flex flex-col gap-2 p-4 rounded-xl border border-l-4"
-        >
-          <div className="flex items-center">
-            {icon}
-            <h2 className="text-xl font-semibold">{title}</h2>
-          </div>
+      {list.map(
+        ({ title, desc, icon, url, urlName, isExternal, isDisabled }) => (
+          <div
+            key={title}
+            className="lg:p-6 flex flex-col gap-2 p-4 rounded-xl border border-l-4"
+          >
+            <div className="flex items-center">
+              {icon}
+              <h2 className="text-xl font-semibold">{title}</h2>
+            </div>
 
-          <p className={TEXT_CLASS}>{desc}</p>
-          {isExternal ? (
-            <a href={url} className="text-purple-600 mt-auto" target="_blank">
-              {urlName}
-              <MoveUpRight className="ml-2 inline" size={16} />
-            </a>
-          ) : (
-            <Link href={url} className="text-purple-600 mt-auto">
-              {urlName}
-            </Link>
-          )}
-        </div>
-      ))}
+            <p className={TEXT_CLASS}>{desc}</p>
+            {isDisabled ? (
```

Comment:
we try not to use nested ternary operators (`a ? b ? c : d : e`), but here might be fine, if it's temporary

Reply:
ah, @truemiller you were faster :D